### DISCUSSION TITLE
Improve export of Goals details

### DISCUSSION THREAD

**Discussion by rvlak** [plausible/analytics#1547] [Anything Else]
Dear folks, I really love the new export feature that was introduced on 15 November. Thanks so much for adding this!

There's one thing that I'm still dearly missing, which is the ability to export conversions for a particular goal (e.g. to show the number of 'Download' conversions for each Download file). I know I can filter the dashboard on a goal and then export the results, but the 'pages.csv' export file is based on conversions by Page, not conversions by Download file (PDF). If there are several Downloads on a single page, I can't see which one converted in the export file pages.csv. And if visitors download a particular PDF directly from the homepage news grid at [https://www.somo.nl/](https://www.somo.nl/), it would be logged as a conversion for the homepage, and I would not have a clue which file they downloaded. The same goes for archive pages (e.g. [https://www.somo.nl/topic/climate-justice/)](https://www.somo.nl/topic/climate-justice/)).

By the way, the exact conversion numbers for each Download file are correctly shown in the 'Goals' section of the dashboard, but for some reason that section isn't included in the export zip, nor can it be opened in a 'Details' view and then copy-pasted into a spreadsheet. I'd prefer to have 'Goals' included in the export zip though, to prevent truncation of longer URLs and make it easier to compile our reports.

Hope this is something that could be added!

**Comment by ukutaht** [comment]
Hey, thanks for the report.

Do you have a `prop_breakdown.csv` file in your export? It should have the custom prop information you're looking for

**Comment by rvlak** [comment]
Hi @ukutaht, thanks for your fast response! I did not know I had to look at this file, that is great news. It contains all the information I need! Thanks again, much appreciated.
